src/Core/Addon/Module/ModuleManager.php

```diff
@@ -615,7 +627,9 @@ public function enableMobile($name)
             );
         }
 
-        $this->clearCache();
+        if ($result && $this->actionParams->get('cacheClearEnabled', true)) {
```

Comment:
This could be a function to remove duplicate.

```php
function checkForCacheAndClearYouKnowItBro($result) {
  if ($result && $this->actionParams->get('cacheClearEnabled', true)) {
    $this->clearCache();
  }
} 
```

Reply:
yep,  good idea

Reply:
done